### PR TITLE
Make nyt clone

### PR DESCRIPTION
**Pull request by vitorelourenco**
Linter checks were added to GitHub Actions
reset.css file included
The structure and content of the NYT page was laid out in html.
The images were downloaded from the website and the SVG icons were copied from the source when available.
The fonts used in the clone are either georgia or the browsers default since the original fonts aren't available for download. 
The header and central column containing the main content of the page were positioned using display: flex.
The Editor's Pick section was placed to the right using float:right and the <aside> tag. This section is currently contained by a div.
The middle section containing links to other articles (between the page's footer and the main content) was built using display: grid.
The bottom section was also built using display: grid , the links weren't added to the respective hrefs of the anchors.
The nav element was fixed to the top and it's elements positioned with display:flex
The fixed-footer element was fixed to the bottom and it's element is positioned using display:flex
A README.md file was written using Microverse's template and the feature branch was deployed. The link to the live version can be found in the README.md file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,727 @@
...
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+        <nav id="nav-bar">
+            <div id="nav-left">
+                <img class="padding-left" src="assets/icons/baseline_menu_black_18dp.png" alt="menu-icon">
+                <img class="padding-left" src="assets/icons/baseline_search_black_18dp.png" alt="menu-icon">
+                <strong class="padding-left">SPACE & COSMOS</strong>
+            </div>
+            <div id="nav-center">
```
Comment (wathigo) [inline index.html:17]: **REQUIRED**
The NY Times logo is a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,727 @@
...
+            </div>
+        </main>
+        <div id="grid-other-articles">
+            <div class="grid-left-subgrid">
+                <div class="grid-box">
+                    <div class="inner-grid">
+                        <div>
+                            <img src="assets/icons/asneeded.png" alt="as needed icon">
+                            <img class="float-right" src="assets/images/plusicon.webp" alt="more icon">
+                        </div>
```
Comment (wathigo) [inline index.html:493]: **REQUIRED**
It uses an anchor tag, any valid value e.g. "#" is okay for it's `href` attribute, to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this project.
Let's make a few changes before we move on. See the comments :point_down: 

**After making the changes, submit another request for review.**

**Happy Coding :technologist: 
Simon Wathigo**

File: index.html (hunk 1 above)
Comment (vitorelourenco) [reply index.html:17]: change made and pushed

File: index.html (hunk 2 above)
Comment (vitorelourenco) [reply index.html]: requested changes were made to this block of code plus the two other blocks following it. changes pushed

**Review by blackpintz — CHANGES_REQUESTED** [review]
# Status: Changes Required 🛑 

## Hello team 👋,

**Great job implementing the requested changes 👏.**

### Highlights.

- **The page looks like the original.**

### Mandatory requirements

- **The new york times logo at the footer should be a link as well.**
- **Wrap your footer section in a `footer` tag instead of `div` tag.**

### Happy coding 🥂.

**You can reach me on slack (Rose Wanjohi).**

**Comment by vitorelourenco** [comment]
> # Status: Changes Required stop_sign
> ## Hello team wave,
> 
> **Great job implementing the requested changes clap.**
> ### Highlights.
> 
>     * **The page looks like the original.**
> 
> 
> ### Mandatory requirements
> 
>     * **The new york times logo at the footer should be a link as well.**
> 
>     * **Wrap your footer section in a `footer` tag instead of `div` tag.**
> 
> 
> ### Happy coding clinking_glasses.
> 
> **You can reach me on slack (Rose Wanjohi).**

changed <div> to <footer> and made the NYT footer logo into a link

**Review by enaburekhan — APPROVED** [review]
# Status: Project Approved:heavy_check_mark:
Hi Vitor!
Great job on implementing all the changes requested by the previous Tse:clap::clap:. The project looks exactly like the original. Keep up the good work:thumbsup:
Move on with your next task:walking:.

You can drop your questions:point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy Coding:blush:
Eric Enaburekhan

**Pull request merged**